ansible/deploy.yml

```diff
@@ -213,7 +213,7 @@
 
       - name: API health check
         uri:
-          url: "{{ api_base_uri }}/top"
+          url: "{{ api_base_uri }}/blocks/top"
```

Comment:
I see this playbook called as deployment in https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/.circleci/config.yml#L159

Reply:
I see infrastructure Docker image fetched by epoch repo https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/.circleci/config.yml#L19

Reply:
I see push of infrastructure Docker image as job in master https://github.com/aeternity/infrastructure/blob/dc2927378b2b5a45cdc4ec52ea45ca29eaa8be88/.circleci/config.yml#L199